projects/vdk-core/src/vdk/internal/builtin_plugins/run/cli_run.py

```diff
@@ -143,14 +144,41 @@ def __log_exec_result(self, execution_result: ExecutionResult) -> None:
 
     @staticmethod
     def __log_short_exec_result(execution_result):
```

Comment:
Add type hints please if you can even though it's not introduced now

 I must have missed to comment on that in previous review.